### ISSUE TITLE
Page not found error when trying to download a pdf from

### ISSUE THREAD

**Issue by ChemiKyle** [grezniczek/redcap_pdf_actiontags#2]
Hi @grezniczek 

Similar to Issue #1, I'm getting a relative path error when trying to download a pdf of an instrument; instead of a redirect it's a PHP error. It is occurring from the "Online Designer" tab, clicking the icon for "View PDF" results in the following error:

```
Fatal error: Uncaught Exception: The specified page does not exist for this module. /var/www.backup/redcap-20190610T1102/modules/redcap_pdf_actiontags_v1.1.1/demographics.php in /var/www.backup/redcap-20190610T1102/redcap_v9.0.3/ExternalModules/index.php:66 Stack trace: #0 {main} thrown in /var/www.backup/redcap-20190610T1102/redcap_v9.0.3/ExternalModules/index.php on line 66
REDCap crashed due to an unexpected PHP fatal error!

Error message: Uncaught Exception: The specified page does not exist for this module. /var/www.backup/redcap-20190610T1102/modules/redcap_pdf_actiontags_v1.1.1/demographics.php in /var/www.backup/redcap-20190610T1102/redcap_v9.0.3/ExternalModules/index.php:66 Stack trace: #0 {main} thrown
File: /var/www.backup/redcap-20190610T1102/redcap_v9.0.3/ExternalModules/index.php
Line: 66
```

The url is:
```
http://redcap.test/redcap/redcap_v9.0.3/ExternalModules/?prefix=redcap_pdf_actiontags&page=pdf&pid=17&&page=demographics
```

~~I'm going to try finding if there is somewhere to recreate what you did in commit 99c7ed94295cf6c278d4a2f178556021c771c2fc and I'll make a PR if it works.~~

I see that the JS is recognizing, but not properly replacing, the link for the icon. Here is the url the link points to, in case it helps you solve it faster:
```
http://redcap.test/redcap/redcap_v9.0.3/ExternalModules/?prefix=redcap_pdf_actiontags&page=pdf&pid=17&&page=demographics
```

**Comment by grezniczek** [comment]
Hi @ChemiKyle , thanks for reporting this. 
It seems I totally missed the Designer as a place where links to PDF/index.php exist .... and thus, the replacement with a link to the module's pdf.php is wrong. 
I will put out a fix momentarily. Thanks.

**Comment by grezniczek** [comment]
Ok. Fix is out (v1.1.2) and submitted to the EM Repo.
